src/@utils/svg.ts

```diff
@@ -162,7 +162,8 @@ export const isSVGProperty = (property: string): property is SVGProperties => {
         'fillOpacity',
         'strokeColor',
         'strokeOpacity',
-        'strokeWidth'
+        'strokeWidth',
+        'className'
```

Comment:
The name of this method is also confusing. Remove the `className` property from here, this is only used to check properties that can be animated.